ci.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -eu
```

Comment:
Could you explain the meaning of this?

Reply:
this will abort the ci.sh script when one of the sub commands will fail. otherwise, when the first gradle call will report an error, the script will ignore this and call the next gradle action.